recipes-browser/cog/cog_0.1.1.bb

```diff
@@ -1,6 +1,7 @@
 require cog.inc
 
-SRC_URI = "https://github.com/Igalia/${PN}/archive/v${PV}.tar.gz"
+SRC_URI = "https://github.com/Igalia/${PN}/archive/v${PV}.tar.gz \
+          "
```

Comment:
Probably you want to remove this change from the commit (it is not really needed), and only leave the file rename.

Reply:
Changed